index.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
-module.exports = function (a, b) {
+module.exports = function (a, b, minComp) {
 	if (!Buffer.isBuffer(a) || !Buffer.isBuffer(b)) {
 		throw new TypeError('Arguments must be Buffers');
 	}
 
+	var len = Math.max(a.length, b.length, minComp || 0);
```

Comment:
But we may do:
```js
	if (a.length != b.length) {
		throw new TypeError('Arguments must be Buffers of same length');
	}
```
I think this is also a valid behavior for a _constant_ equals.